src/sentry/static/sentry/app/views/settings/organizationRepositories/addRepositoryLink.jsx

```diff
@@ -108,13 +108,29 @@ class AddRepositoryLink extends PluginComponentBase {
   renderForm() {
     let errors = this.state.error.errors || {};
     let provider = this.props.provider;
+    let hasIntegration = true;
+    provider.config.forEach(field => {
+      if (field.initial == '') hasIntegration = false;
```

Comment:
@MeredithAnya looks like your checking if any field in config doesn't have an initial then give them the empty message. Am I reading that correctly? Would it be better to pass a flag from the serializer? Just wondered if there's a case where something might be purposefully empty. Feel free to let me know if I'm misinterpreting stuff.

Reply:
I could be wrong but if there just isn't an initial field, it should be undefined. I want to know if we passed in an empty choices option (meaning we couldn't find an integration)